pkg/utils/binpacking/binpacking.go

```diff
@@ -40,3 +41,69 @@ func (r ByResourcesRequested) Less(a, b int) bool {
 	}
 	return resourcePodA.Cpu().Cmp(*resourcePodB.Cpu()) == -1
 }
+
+func MergeResources(resources ...v1.ResourceList) v1.ResourceList {
```

Comment:
This is almost the same as scheduling.GetResources(), can we dedup?

Reply:
GetResources() is accepting a set of podSpec and returning the `ResourceList` for all the specs. Here we are combining a list of `ResourceList` into one. 
We would still need a function that will accept podSpecs and loop thorough all pod.Containers to sum the Quantities.